### ISSUE TITLE
Historical configuration for CLM6

### ISSUE THREAD

**Issue by wwieder** [ESCOMP/CTSM#2570]
In our meeting today there was discussion on creating land use time series for historical simulations with CLM6.  We want to be able to run full historical simulations "out of the box" through 2022 (last year of CRUJRA inputa data).  This will avoid anoying restarts in 2015 to continue historical simualtions.  Most of the discussion was on the land use time series, but I think this also applies to datm.streams that we're reading in.

Regarding the landuse timeseries, we discussed several options:

1. Remake the landuse timeseries to cover 1850-2022 with existing **CTSM5.2** historical datasets and SSP2-4.5. 
  - Note, this is where the continuous ssp extensions can also be set up for historical simulations with datm.streams 
2. Create the landuse timeseries as above, but with the **CTSM5.3** PR, #2500 
3. Create the landuse timeseries using the updated LUH2 data from **TRENDY**, which has been updated from the CMIP6 data
4. Wait for the official CMIP7 landuse data to come out, which will likely be the same as # 3.

Here are my questions:
- Quickest path forward would be option 1, we just need to remake the data that's already on hand. @slevis-lmwg thinks this could be done pretty efficiently before the summer code freeze.
- Options 3 & 4 will likely be what we want to use for the CESM3 release, but is it urgent to do these before the summer code freeze?
- If we ultimately want options 3/4, what is gained in the 5.3 work at all?  Is it just the high res capabilities?

@lawrencepj1 and @dlawrenncar can you weigh in here on the best path forward.

**Comment by ekluzek** [comment]
@wwieder option 1 is already covered as we use the SSP2-4.5 landuae time series files for historical simulations.

**Comment by lawrencepj1** [comment]
Hi @wwieder 

I think the idea for option 3 is that it will be very close to CMIP7 land use and consistent with the next version of Trendy to be run this summer so allows us to progress without waiting for CMIP7 which has a pretty open timeframe at the moment. Additionally this option has the updated PFT and CFT distributions which will be beneficial for the high resolution data.

Peter

**Comment by dlawrenncar** [comment]
I think that Options 3 / 4 are what we will need for the release.  I don't
think we expect big changes from what is in CTSM5.2, so I think it might be
ok to just use what we are doing now (e.g., extending LUH2 with SSP2-4.5 to
get to 2022) for any historical test simulations that are being done in
land-only or coupled configurations.  But, we should be pushing towards
Option 3, superseded by Option 4 if beta CMIP forcing data becomes
available.  I don't think this is required by the code freeze.  Changes
from tuning will be much larger than changes due to these small land use
forcing changes, but for the sake of simplicity, the earlier we can get
option 3 in place, the better.   But, it's not a code freeze deadline issue.

**Comment by wwieder** [comment]
From discussions at the CESM workshop it sounds like @dlawrenncar and @lawrencepj1 suggested migrating to CTSM5.3 datasets should happen concurrently with the switch to the TRENDY LULCC land use time series.  This should also happen before we start trying to evaluate/calibrate historical simulations in CLM or CESM3.  I'm still a bit uncertain on timing, this may not be critical before the July 31 deadline, but maybe should be soon thereafter?

**Comment by slevis-lmwg** [comment]
@lawrencepj1 as you and I confirmed last week, our next pft/landuse meeting (includes @ekluzek and @mvdebolskiy) is scheduled for July 17th. Regardless, pls let me know when you have the new raw datasets ready for me to test with mksurfdata_esmf and a new simulation.

**Comment by slevis-lmwg** [comment]
@slevis-lmwg @lawrencepj1 @samsrabin @ekluzek @linniahawkins discussed next steps for ctsm53.
@samsrabin took notes here:
https://docs.google.com/document/d/1-2LEjIhbs4j4NoythdLqdJ9A1pcNt4UJ_tyda2rrQWc

A PR documenting this work is already open: #2500
